content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -46,50 +63,50 @@ git init
 
 Alternatively, you can [create a repo on GitHub](https://help.github.com/articles/create-a-repo/) to version and share your work. After setting it up, [clone the repo](https://help.github.com/articles/cloning-a-repository/) to your local machine and `cd` into that directory.
 
-#### 2. Create a new `package.json` file for your project.
+#### 2. Create a new `package.json` file for your project
 
 Use the following template, which includes:
 
-* RippleAPI itself (`ripple-lib`)
-* Babel (`babel-cli`)
-* The ECMAScript 6 presets for Babel (`babel-preset-es2015`)
-* (Optional) [ESLint](http://eslint.org/) (`eslint`) for checking code quality.
+- RippleAPI itself (`ripple-lib`)
+- (Optional) [ESLint](http://eslint.org/) (`eslint`) for checking code quality.
 
 ```
 {% include '_code-samples/rippleapi_quickstart/package.json' %}
 ```
 
+#### 3. Use Yarn to install the dependencies
 
-#### 3. Use NPM to install the dependencies.
+Use Yarn to install the dependencies defined in the `package.json` file you created for your project.
 
 ```
-npm install
+yarn
 ```
 
-This automatically installs all the dependencies defined in the `package.json` into the local folder `node_modules/`. (We recommend _not_ using `npm -g` to install the dependencies globally.)
+This installs the dependencies into the local folder `node_modules/`.
 
-The install process may take a while and end with a few warnings. You may safely ignore the following warnings:
+The install process may end with an `eslint` warning. You may safely ignore the following warning: ***TODO: necessary to mention? Existing doc surfaced NPM warnings.***
 
 ```
-npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
-npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.6
-npm WARN ajv@1.4.10 requires a peer of ajv-i18n@0.1.x but none was installed.
+warning eslint > file-entry-cache > flat-cache > circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
 ```
 
-# First RippleAPI Script
+
+
+## Create Your First RippleAPI Script
```

Comment:
Bumping the hierarchy down 1 level means that some items no longer in the table of contents. I think having a second "h1" is better than doing that.

It might also be OK to split this into two separate pages entirely.

Reply:
Changed them back to H1s. I get it 👍 !